doc/howto/move_instances.md

```diff
@@ -63,6 +63,13 @@ To do so, ensure the following configuration:
 If you are using a shared storage pool to back your instance like Ceph RBD, you don't need to set the [`size.state`](devices-disk) property.
 ```
 
+The default value for {config:option}`instance-migration:migration.stateful` was introduced with the {config:option}`server-miscellaneous:instances.migration.stateful` introduced with the `stateful_migration_default` API extension and applies only to new instances. Existing instances won't be affected by this change.
```

Comment:
@ru-fu do we routinely mention API extensions in our general user docs, or is it assumed the docs for latest apply to the latest version of LXD only?

Reply:
The docs apply only to the latest version of LXD. However, in this case, the latest version might be running instances that have been created earlier, so this information is relevant even for the latest version.
It's not something we do routinely, but imo we should point it out here.

Reply:
thanks!